content/docs/command-reference/metrics/index.md

```diff
@@ -1,47 +1,104 @@
 # metrics
 
-A set of commands to add, manage, collect, and display project metrics:
-[add](/doc/command-reference/metrics/add),
-[show](/doc/command-reference/metrics/show),
-[diff](/doc/command-reference/metrics/diff),
-[modify](/doc/command-reference/metrics/modify), and
-[remove](/doc/command-reference/metrics/remove).
+A set of commands to manage, collect, and display _project metrics_:
+[show](/doc/command-reference/metrics/show), and
+[diff](/doc/command-reference/metrics/diff).
 
 ## Synopsis
 
 ```usage
 usage: dvc metrics [-h] [-q | -v]
-                   {show,add,modify,remove,diff} ...
+                   {show,diff} ...
 
 positional arguments:
   COMMAND
     show                Print metrics, with optional formatting.
-    add                 Mark a DVC-tracked file as a metric.
-    modify              Modify metric default formatting.
-    remove              Remove metric mark on a DVC-tracked file.
     diff                Show changes in metrics between commits
 ```
 
 ## Description
 
-In order to track metrics associated to machine learning experiments, DVC has
-the ability to mark a certain stage <abbr>outputs</abbr> as files containing
-metrics to track (see the `--metrics` option of `dvc run`). Metrics are
-project-specific floating-point values e.g. `AUC`, `ROC`, etc.
+In order to track the basic performance of machine learning experiments, DVC has
+the ability to mark a certain stage <abbr>outputs</abbr> _scalar metrics_ to
+track.
 
-Supported file formats: JSON. Metrics can be organized in a tree hierarchy in a
-JSON file. DVC addresses the metrics by the tree path.
+These metrics are project-specific floating-point or integer values e.g. AUC,
+ROC, _false positives_, etc. They can be defined with the `-m` (`--metrics`) and
+`-M` (`--metrics-no-cache`) options of `dvc run`.
 
-DVC itself does not ascribe any specific meaning for these numbers. Usually
-these numbers are produced by the model training or model evaluation code and
-serve as a way to compare and pick the best performing experiment.
+### Types of metrics
```

Comment:
this should go before description

Reply:
I didn't think you meant that literally earlier since it breaks our standard cmd ref structure. But OK, did it now in 3b898b37 (included also continuous and scalar terminology update discussed in Slack).

WDYT? I'm not so sure about this one...

Reply:
> standard cmd ref structure.

we should not be following or trying to preserve it a cost of actual meaning and natural flow of the text. I don't know if this should go before or should be part of description, but introducing types and explaining the difference should be before explaining a particular type.

Reply:
Would be tricky putting it in the Description before the basic explanation unless we remove the **Types of metrics** header and just make it into the first paragraph.

But for now I think it's fine actually, since this change only affects the index.md of both metrics and plots, which is kind of a placeholder cmd ref anyway. Other indices are used to explain concepts more than as cmd refs also, like for cache and remote 🙂 so we can be more flexible about these in terms of doc structure going fwd.